vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
+COMPONENTS_FILEPATH=/opt/azure/components.json
 
 testFilesDownloaded() {
   test="testFilesDownloaded"
   echo "$test:Start"
-  filesToDownload=$1
-
-  filesToDownload=$(echo $filesToDownload | jq -r ".[]" | jq . --monochrome-output --compact-output)
+  filesToDownload=$(cat $COMPONENTS_FILEPATH)
```

Comment:
can you also convert this to:
filesToDownload=$(jq .DownloadFiles[] --monochrome-output --compact-output < $COMPONENTS_FILEPATH)